ansible/group_vars/common.yml

```diff
@@ -22,32 +22,56 @@ dummy:
 # GENERAL #
 ###########
 
-opensds_release: v0.1.4 # The version should be at least v0.1.4.
-nbp_release: v0.1.0 # The version should be at least v0.1.0.
-
-# These fields are not suggested to be modified
-opensds_download_url: https://github.com/opensds/opensds/releases/download/{{ opensds_release }}/opensds-{{ opensds_release }}-linux-amd64.tar.gz
-opensds_tarball_url: /opt/opensds-{{ opensds_release }}-linux-amd64.tar.gz
-opensds_dir: /opt/opensds-{{ opensds_release }}-linux-amd64
-nbp_download_url: https://github.com/opensds/nbp/releases/download/{{ nbp_release }}/opensds-k8s-{{ nbp_release }}-linux-amd64.tar.gz
-nbp_tarball_url: /opt/opensds-k8s-{{ nbp_release }}-linux-amd64.tar.gz
-nbp_dir: /opt/opensds-k8s-{{ nbp_release }}-linux-amd64
+# This field indicates which way user prefers to install, currently support 'repository', 'release' and 'container'
+install_from: repository
 
+# These fields are not suggested to be modifie
+opensds_work_dir: /opt/opensds-linux-amd64
+nbp_work_dir: /opt/opensds-k8s-linux-amd64
 opensds_config_dir: /etc/opensds
 opensds_log_dir: /var/log/opensds
 
+##############
+# REPOSITORY #
+##############
+
+# If user specifies intalling from repository,then he can choose the specific branch
+opensds_repository: development
+nbp_repository: master
+
+# These fields are not suggested to be modified
+opensds_remote_url: https://github.com/opensds/opensds.git
+nbp_remote_url: https://github.com/opensds/nbp.git
+
 
 ###########
-# PLUGIN  #
+# RELEASE #
 ###########
 
-nbp_plugin_type: standalone # standalone is the default integration way, but you can change it to 'csi', 'flexvolume'
+# If user specifies intalling from release,then he can choose the specific version
+opensds_release: v0.1.7 # The version should be at least v0.1.4
+nbp_release: v0.1.0 # The version should be at least v0.1.0
 
-flexvolume_plugin_dir: /usr/libexec/kubernetes/kubelet-plugins/volume/exec/opensds.io~opensds
+# These fields are not suggested to be modified
+opensds_download_url: https://github.com/opensds/opensds/releases/download/{{ opensds_release }}/opensds-{{ opensds_release }}-linux-amd64.tar.gz
+opensds_tarball_dir: /tmp/opensds-{{ opensds_release }}-linux-amd64
+nbp_download_url: https://github.com/opensds/nbp/releases/download/{{ nbp_release }}/opensds-k8s-{{ nbp_release }}-linux-amd64.tar.gz
+nbp_tarball_dir: /tmp/opensds-k8s-{{ nbp_release }}-linux-amd64
+
+
+#############
+# CONTAINER #
+#############
+
+container_enabled: false
```

Comment:
Does this option take any effect?  If set to true, what does it do?

Reply:
If it's set true, then the controller, dock and etcd service would run in container.